gcs/deleteFile/proto/deleteFile.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+//
+// The DeleteFileProbe proto describes the information necessary for a probe to delete a
+// file in GCS.
+
+syntax = "proto2";
+
+package gcs.deletefile;
```

Comment:
Shouldn't this be a package under the "hermes" namespace?

Reply:
We discussed this in our pod meeting, and this has been changed to `hermes.deletefile`.